src/platform_impl/linux/wayland/window/mod.rs

```diff
@@ -260,6 +260,9 @@ impl Window {
             window_requests.clone(),
         );
 
+        // Set opaque region.
+        window_handle.set_opaque(!attributes.transparent);
+
```

Comment:
Add changelog entry about this + add it to the list of platforms `WindowBuilder::with_transparent` supports

Reply:
Do you mean that opaque region is now set on Wayland? I'm not quite sure that it should be stated in a changelog tbh...

Or do you mean something else? Like it shouldn't fix or affect the users in any way, since it's just a hint and the window was considered transparent.

Reply:
If it's just a hint then no problem

Reply:
Yeah, most apps don't even bother to set it.